ocs_ci/ocs/amq.py

```diff
@@ -446,7 +447,170 @@ def run_in_bg(self, namespace=constants.AMQ_NAMESPACE, value='10000', since_time
+    def run_amq_benchmark(
+        self, name="benchmark", kafka_namespace=constants.AMQ_NAMESPACE,
+        tiller_namespace="tiller", num_of_clients=8, worker=None, timeout=1200,
+        workload_name='1-topic-16-partition-1Kb', topics=1,
+        partitions_per_topic=16, message_size=1024, payload_file="payload/payload-1Kb.data",
+        subscriptions_per_topic=1, consumer_per_subscription=3, producers_per_topic=3,
+        producer_rate=50000, consumer_backlog_sizegb=0, test_duration_minutes=15
+    ):
+        """
+        Run benchmark pod and get the results
+
+        Args:
+            name (str): Name of the benchmark pod
+            kafka_namespace (str): Namespace where kafka cluster created
+            tiller_namespace (str): Namespace where tiller pod needs to be created
+            num_of_clients (int): Number of clients to be created
+            worker (str) : Loads to create on workloads separated with commas
+                e.g http://benchmark-worker-0.benchmark-worker:8080,
+                http://benchmark-worker-1.benchmark-worker:8080
+            timeout (int): Time to complete the run
+            workload_name (str): Name of the workloads
+            topics (int): Number of topics created
+            partitions_per_topic (int): Number of partitions per topic
+            message_size (int): Message size
+            payload_file (str): Load to run on workload
+            subscriptions_per_topic (int): Number of subscriptions per topic
+            consumer_per_subscription (int): Number of consumers per subscription
+            producers_per_topic (int): Number of producers per topic
+            producer_rate (int): Producer rate
+            consumer_backlog_sizegb (int): Size of block in gb
+            test_duration_minutes (int): Time to run the workloads
+
+        Return:
+            result (json): Returns the test result
+
+        """
+
+        # Namespace for to helm/tiller
+        try:
+            self.create_namespace(tiller_namespace)
+        except CommandFailed as ef:
+            if f'project.project.openshift.io "{tiller_namespace}" already exists' not in str(ef):
+                raise ef
+
+        # Create rbac file
+        try:
+            sa_tiller = list(templating.load_yaml(constants.AMQ_RBAC_YAML, multi_document=True))
+            sa_tiller[0]["metadata"]["namespace"] = tiller_namespace
+            sa_tiller[1]["subjects"][0]["namespace"] = tiller_namespace
+            self.sa_tiller = OCS(**sa_tiller[0])
+            self.crb_tiller = OCS(**sa_tiller[1])
+            self.sa_tiller.create()
+            self.crb_tiller.create()
+        except(CommandFailed, CalledProcessError) as cf:
+            log.error('Failed during creation of service account tiller')
+            raise cf
+
+        # Install helm cli (version v2.16.0 as we need tiller component)
+        # And create tiller pods
+        url = "https://get.helm.sh/helm-v2.16.1-linux-amd64.tar.gz"
```

Comment:
what if the version of helm changes in future and we have to use most recent version? better not to hardcode it. check

Reply:
From the doc https://gist.github.com/ksingh7/31da820f522050ab9e0a7c1599d13d6b, it is the stable one so used the same version

Reply:
As discussed, will take it in next PR